addons/binding/org.openhab.binding.synopanalyzer/README.md

```diff
@@ -20,15 +20,14 @@ Synop message are typically updated every hour.
 
 The weather information that is retrieved is available as these channels:
 
-| Channel Type ID     | Item Type | Description                                |
-|---------------------|-----------|--------------------------------------------|
-| temperature         | Number    | The current temperature in degrees Celsius |
-| pressure            | Number    | The current pressure in millibar (mb)      |
-| wind-angle          | Number    | Wind angle in degrees                      |
+| Channel Type ID     | Item Type          | Description                                |
```

Comment:
Please apply the Markdown Table formatter: https://www.tablesgenerator.com/markdown_tables